product_docs/docs/ark/3.4/ark_admin/04_ark_registering_with_pem.mdx

```diff
@@ -43,28 +42,26 @@ If you select `ENABLED`:
 
 -   Any Amazon Role or Azure Group that is accessible by the Ark user that is deploying the console is added to the PEM server as a group role. The PEM role created for the project, role, or group is not a login role. To simplify management, each role that represents a project, role, or group is also a member of the PEM `ark-team` group role.
 
-![*The PEM Group Role dialog*](images/ark_registering_with_pem_group_role_dialog.png)
+![\*The PEM Group Role dialog\*](images/ark_registering_with_pem_group_role_dialog.png)
```

Comment:
Introduces backslash characters inside image ALT tags

Reply:
So, this is strickly-speaking unnecessary: the spec allows formatting in alt text, but recommends that they be ignored (stripped during render). The parser here apparently attempts to preserve them as *text*, under the assumption that they were intended to be literal. 

Of course, they weren't - this was a caption, not alt text, and the asterisks were intended to italicize the caption. So... We probably need two decisions here:

- Do we want to allow formatting in captions? (we could, uh, just style captions however we want) - if so, they should be preserved so that they can be re-rendered properly by whatever.
- CAN we render formatting in captions? (this is a question for @epbarger ) - if not, I should probably just strip this entirely.

Meanwhile, I'll take a look at what's there; I should be able to block escaping, but may not want to if we're gonna just strip these.

Reply:
Ok, these are no longer escaped. We can decide down the road what we want to do with them.